### DISCUSSION TITLE
Is there a way to have parcel NOT put multiple entrypoints into separate folders in the dist-dir?

### DISCUSSION THREAD

**Discussion by adrianocr** [parcel-bundler/parcel#8690] [Q&A] [answered]
If I do something like:

`parcel watch ['./src/js/dir-1/*.js', './src/js/other-dir/*js'] --dist-dir ./assets` 

or

`parcel watch ./src/js/**/*.js --dist-dir ./assets` 

parcel will output the js files in the ./assets dir as the following: `/assets/dir-1/filename.js` and `/assets/other-dir/other-filename.js`. 

When in reality I'm interested in /assets/filename.js and /assets/other-filename.js. 

Is there a way to get parcel to "flatten" the way it outputs files into the dist-dir instead of sub-directing them per the directory they are in?

**Comment by devongovett** [comment, marked as answer]
The problem with flattening is that the two directories could have files of the same name causing conflicts.

But you could do this with a custom Namer plugin.

**Reply by adrianocr** [reply]
Thank you, I was able to resolve it with https://www.npmjs.com/package/parcel-namer-custom since I know for a fact my filenames will never conflict
